src/client/theme-default/styles/vars.css

```diff
@@ -147,8 +147,14 @@
 :root {
   --vp-font-family-base: 'Inter var experimental', 'Inter var', -apple-system,
     BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell,
-    'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif;
-  --vp-font-family-mono: Menlo, Monaco, Consolas, 'Courier New', monospace;
+    'Fira Sans', 'Droid Sans', 'Helvetica Neue', 'Helvetica', 'Noto Sans',
```

Comment:
I wonder if it's worth listing so many sans-serif fonts here.

Is listing 13 CJK fonts an improvement over shorter options, such as:

- Just defaulting to `sans-serif`, so: `--vp-font-family-base: Inter var experimental, Inter var, sans-serif;`
- Or defaulting to `system-ui` first, then the generic `sans-serif`: `--vp-font-family-base: Inter var experimental, Inter var, system-ui, sans-serif;`

If there is a need to specify more fonts because system and browser default fonts for CJK scripts tend to look bad, would there be a way to limit it to just a few options and not ~13 maybe?

Notes:

- Browser support for `system-ui` is good: https://caniuse.com/font-family-system-ui
- `-apple-system` and `BlinkMacSystemFont` are not needed anymore, use `system-ui` instead.

Reply:
The usage of the CJK fonts here is indeed a bit strange. There are several possible problems here:

1. If the web page (or a part of the web page, because a web page can be multilingual) is not properly language tagged, it may cause the wrong font to be applied (like [this example](https://www.w3.org/International/articles/lang-bidi-use-cases/index.en.html#text_processing)).

2. IIRC the Noto CJK fonts have been renamed, but here it only contains the old names.

3. There might be potential issues with `system-ui`. See https://github.com/w3c/csswg-drafts/issues/3658

-----

(See also https://github.com/w3c/clreq/issues/408#issuecomment-1103501409 for related discussions about Chinese fonts.)

Reply:
To avoid this kind of issue, maybe we should just list the Latin fonts here, not the fonts for other scripts.